src/node/chainstate.cpp

```diff
@@ -167,6 +134,93 @@ ChainstateLoadResult LoadChainstate(ChainstateManager& chainman, const CacheSize
     return {ChainstateLoadStatus::SUCCESS, {}};
 }
 
+ChainstateLoadResult LoadChainstate(ChainstateManager& chainman, const CacheSizes& cache_sizes,
+                                    const ChainstateLoadOptions& options)
+{
+    if (!hashAssumeValid.IsNull()) {
+        LogPrintf("Assuming ancestors of block %s have valid signatures.\n", hashAssumeValid.GetHex());
```

Comment:
I think could do a `LookupBlockIndex` to display the block height, as it's more speaking for the node operator.

Reply:
Out of scope for this PR, since this is just code I moved.